### PR TITLE
chore: test against go-ipfs 0.4.13

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by diasdavid** [comment]
A ton of tests is failing when testing js-ipfs-api against go-ipfs@0.4.13. @keks do you happen the know why? Could this be related to cmds changes?

**Comment by keks** [comment]
We only merged the cmds stuff right after 0.4.13, so that can't really be the reason.

**Comment by diasdavid** [comment]
@keks any other HTTP-API changes that came in 0.4.12 or 0.4.13?

**Comment by keks** [comment]
Not sure - 0.4.13 was a hotfix release right after 0.4.12, I think there were very few changes. Do the tests pass for 0.4.12?

**Comment by richardschneider** [comment]
@diasdavid installed 0.4.13 and used my net-http-api client against it.  All tests passed.

**Comment by diasdavid** [comment]
@richardschneider do you make any use of the HTTP Headers in your library? Especially the x-stream and x-chunked?

**Comment by richardschneider** [comment]
@diasdavid I don't use any special headers, except asking for GZIP response.

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "peer-id": "~0.10.3",
     "peer-info": "~0.11.3",
     "promisify-es6": "^1.0.3",
-    "pump": "^2.0.0",
+    "pump": "^1.0.3",
```
Comment (diasdavid) [inline package.json]: @keks @richardschneider this was the culprit! Apologies for taking from your time

**Pull request merged**